src/pages/Landing/components/StudentTicket/History/History.module.scss

```diff
@@ -0,0 +1,127 @@
+.history{
+   
+    @mixin text {
+     
+font-family: "Muli Bold";
```

Comment:
Install `Prettier - Code formatter` extension to format code plz

Reply:
i install both packages before start coding

Reply:
please check new changes

Reply:
I still see the errors when pull your PR 
![image](https://user-images.githubusercontent.com/43199095/111999751-d9578180-8b4f-11eb-815b-b3212a6b9031.png)

Reply:
Did you enable That extensions?

Reply:
how to enable these extensions.... by using this command??
touch .prettierrc

Reply:
https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode
Go to Extension tab on left panel -> click on extension. The link above is to format code.